### DISCUSSION TITLE
Erreur 0x00007FFE47753CAS

### DISCUSSION THREAD

**Discussion by HervePin** [community/community#121311] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Bonjour,
Depuis hier, une fenêtre Windows s'ouvre en indiquant l'erreur "0x00007FFE47753CAS", la mémoire ne peut être lue. En cochant OK l'application KeePssXC se ferme.
Je dispose de la dernière version 2.7.7
Que dois-je faire?
Bien cordialement

**Comment by Akash1134** [comment]
Hi @HervePin ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as this one. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
